docs/reflection-milestone4.md

```diff
@@ -0,0 +1,11 @@
+# Reflection
+
+## Progress of the dashboard
+
+We have successfully implemented most of functionalities deigned for both Dash Python and R World COVID-19 dashboard based on our initial proposal. In the milestone 4, we improved both of our dashboard based on feedback received from our peers and the teaching team. The major functionality implemented so far includes adding scatter points with rolling averages, making the plot and filter description more concise, hovering over a question mark icon for descriptions, containerizing plots for a better visual presentation, adding a footage and data source link, and making legend and axis titles for a human readable format.
+
+There are still areas that we can improve in the future release. As for instance, we still noctied a performance issue in the R version of app when loading the charts, even we have used a smaller dataset and grouped them as one callback functions. It is still under investigation on how to future improve the loading time for the R version. We also hope to include the last updated data stamp in the footer based on original data source. Last but not the least, we hope to filter countries by continents and so audiences can filter data by the country set instead of individuals. In future releases we can also make the app more responsive like to immediately change the overall size of the content and plots based on the browser window size. This helps audiences to assess our dashboard in a different browser size and even on the mobile app.
```

Comment:
```suggestion
There are still areas that we can improve in the future release. As for instance, we still noctied a performance issue in the R version of app when loading the charts, even we have used a smaller dataset and grouped them as one callback functions. It is still under investigation on how to future improve the loading time for the R version. We also hope to include the last updated data stamp in the footer based on original data source. We have implemented filtering countries by continents in Python so audiences can filter data by the country set instead of individuals. However further fine-tuning on the positions of legend labels is needed if a large number of countries are shown. Also, we could implement the continents filter in R once the performance constraint is resolved. In future releases, we can also make the app more responsive like to immediately change the overall size of the content and plots based on the browser window size. This helps audiences to assess our dashboard in a different browser size and even on the mobile app.
```

Reply:
I've expanded a bit on the continent filtering